src/plugins/dropdown_buttons/plugin.js

```diff
@@ -20,16 +20,20 @@ Selectize.define('dropdown_buttons', function (options) {
 	// We can not remove all selected values and keep the disabled ones in one change event.
 	// So we remove all selected values and add the disabled ones again.
 	function selectNoneOptions() {
+		if (self.items.length === 0) return;
+
 		const alreadySelectedAndDisabledItems = self.items.filter(item => {
 			return self.options[item].disabled;
 		})
+		if (self.items.length === alreadySelectedAndDisabledItems.length) return;
+
 		self.clear({silent: true});
 
 		self.addItems(alreadySelectedAndDisabledItems);
 		self.focus();
 
 		self.refreshOptions(true);
-		self.$input.trigger('change');
+		if (alreadySelectedAndDisabledItems.length === 0) self.$input.trigger('change');
```

Comment:
the main fix, other places are early returns if no necessary to continue logic.

Reply:
How do you test to see if it works?

Reply:
you may copy these changes into selectize in node_modules folder, 
or change in package.json github URL to this PR to get selectize from this branch